tests/images/run.sh

```diff
@@ -6,6 +6,9 @@
 ./bin/image-classifier tests/images/imagenet/*.png -image_mode=0to256 -m=zfnet512 $@
 ./bin/image-classifier tests/images/imagenet/*.png -image_mode=0to1 -m=densenet121 $@
 ./bin/image-classifier tests/images/imagenet/*.png -image_mode=0to1 -m=shufflenet $@
+./bin/image-classifier tests/images/mnist/*.png -image_mode=0to1 -m=lenet_mnist $@
+./bin/image-classifier tests/images/imagenet/*.png -image_mode=0to256 -m=inception_v1 $@
+./bin/image-classifier tests/images/imagenet/*.png -image_mode=0to256 -m=bvlc_alexnet $@
```

Comment:
maybe it's worthwhile to mention in the docs about model zoo we have and what models are in the model zoo.